src/plugins/telemetry/server/fetcher.ts

```diff
@@ -99,13 +167,11 @@ export class FetcherTask {
     }
 
     let clusters: EncryptedTelemetryPayload = [];
-    this.isSending = true;
```

Comment:
We no longer need this because we are calling the method via `exhaustMap`, which essentially ensures it's called only once in multiple concurrent emissions (and skips the rest).